### PR TITLE
[Robot] Changes to object manager page object and NPSP.robot keyword

### PR DESCRIPTION
**Pull request by rjanjanam**
Minor changes for the object manager page object and to the NPSP.robot keyword as suggested by Bryan to make it part of the central cumulusci repo

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/resources/ObjectMangerPageObject.py [hunk 1]
```diff
@@ -103,18 +103,18 @@ def create_lookup_field(self,field_name, related):
...
 		   IF the custom field exists it will not create the custom field and exits out of object manager
 		 """
 		search_button = npsp_lex_locators['object_manager']['input'].format("globalQuickfind")
 		self.selenium.wait_until_page_contains_element(search_button,60)
-		self.selenium.get_webelement(search_button).send_keys(field_name)
+		self.selenium.get_webelement(search_button).send_keys(kwargs['Field_Name'])
 		self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
 		time.sleep(1)
-		self.builtin.log(formula)
+		#self.builtin.log(formula)
```
Comment (Safiyab) [inline robot/Cumulus/resources/ObjectMangerPageObject.py:115]: If you don't need this, can you remove this line?

File: robot/Cumulus/resources/ObjectMangerPageObject.py [hunk 2]
```diff
@@ -103,18 +103,18 @@ def create_lookup_field(self,field_name, related):
 												   message="Detail page did not load in 1 min")
 	
 	@capture_screenshot_on_error
-	def create_custom_field(self, type, field_name, related_to=None, formula=None):
+	def create_custom_field(self, **kwargs):
 		"""Ensure that the custom field does not exist prior and Creates a custom field based on type paramenter and the field_name
```
Comment (Safiyab) [inline robot/Cumulus/resources/ObjectMangerPageObject.py:107]: Should this be "based on Field_Type parameter" ??

File: robot/Cumulus/resources/ObjectMangerPageObject.py [hunk 3]
```diff
@@ -103,18 +103,18 @@ def create_lookup_field(self,field_name, related):
...
 		self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
 		time.sleep(1)
-		self.builtin.log(formula)
+		#self.builtin.log(formula)
 		self.salesforce.wait_until_loading_is_complete()
-		search_results = npsp_lex_locators['object_manager']['search_result'].format(field_name)
+		search_results = npsp_lex_locators['object_manager']['search_result'].format(kwargs['Field_Name'])
 		count = len(self.selenium.get_webelements(search_results))
 		if count == 1:
 			return
```
Comment (Safiyab) [inline robot/Cumulus/resources/ObjectMangerPageObject.py:120]: You can avoid an 'if..else' condition here. You can simplify this as:
```
if not count == 1:
   <all your 'else' code will be here>
```

File: robot/Cumulus/resources/ObjectMangerPageObject.py (hunk 2 above)
Comment (skristem) [reply robot/Cumulus/resources/ObjectMangerPageObject.py:107]: May also mention the acceptable keys as they have to be specifically in certain format

Comment (skristem) [reply robot/Cumulus/resources/ObjectMangerPageObject.py:107]: can you name it starting with _ as it is a helper function

**Pull request merged**